gabarit/template_nlp/nlp_project/package_name/monitoring/model_explainer.py

```diff
@@ -140,6 +141,10 @@ def explain_instance(self, content: str, class_or_label_index: Union[None, int,
         Returns:
             (?): An explanation object
         '''
+        # If a numpy object is passed as class_or_label_index we convert it to an object of a builtin type
+        if is_ndarray_convertable(class_or_label_index):
+            class_or_label_index = ndarray_to_builtin_object(class_or_label_index)
```

Comment:
`is_ndarray_convertable` is already called in `ndarray_to_builtin_object`
we should either not check if convertable here, or in the `ndarray_to_builtin_object` function